GNUmakefile

```diff
@@ -50,16 +50,20 @@ cov:
 	open /tmp/coverage.html
 
 test: dev
-	go test -tags "$(GOTAGS)" -i ./...
-	go test -tags "$(GOTAGS)" -run '^$$' ./... > /dev/null
```

Comment:
Minor thing, but this what I am seeing locally:
```
# github.com/hashicorp/consul/command                                                                           │--- PASS: TestACLPolicy_Bad_Policy (0.00s)
command/keygen_test.go:21:22: missing ',' before newline in composite literal                                   │PASS
FAIL    github.com/hashicorp/consul/command [setup failed] 
<keeps hanging here for a while, eventually I killed it...>
```
In the master branch after the above compilation error it immediately exits with a 1.